bgpd/bgp_open.c

```diff
@@ -609,7 +609,7 @@ static int bgp_capability_llgr(struct peer *peer,
 
 	SET_FLAG(peer->cap, PEER_CAP_LLGR_RCV);
 
-	while (stream_get_getp(s) + 4 <= end) {
+	while (stream_get_getp(s) + 7 <= end) {
```

Comment:
Can we turn this into a define?  and comprae it to that?

Reply:
Fixed.